### ISSUE TITLE
[CA] Handle keyword arguments in "marca" template

### ISSUE THREAD

**Issue by BoboTiG** [BoboTiG/ebook-reader-dict#1521]
- Wiktionary page: https://ca.wiktionary.org/wiki/4_de_8

Wikicode:
```
{{marca|ca|castells|sort=0}}
```

Output:
```
<i>(castells, sort=0)</i>
```

Expected:
```
<i>(castells)</i>
```

The expected output is not the same as on the Wiktionary, but for a first fix it will be good enough.

---

Model link, if any: https://ca.wiktionary.org/wiki/Plantilla:marca
All arguments can be found here: https://ca.wiktionary.org/wiki/M%C3%B2dul:etiquetes#L-87

**Comment by BoboTiG** [comment]
Another more complex use case: https://ca.wiktionary.org/wiki/Ianquil%C3%A0ndia

**Comment by BoboTiG** [comment]
https://ca.wiktionary.org/wiki/Oxirrinc

**Comment by BoboTiG** [comment]
https://ca.wiktionary.org/wiki/acusador

**Comment by BoboTiG** [comment]
Another fix: f253e6a3fd8e0d03c58de58afdb8e1d2c9e25362
